src/exercise/server/services/itemController.js

```diff
@@ -0,0 +1,40 @@
+// The ItemManager should go here. Remember that you have to export it.
+
+const ItemManager = require('./ItemManager');
+
+const itemManager = new ItemManager;
+
+async function getTasks(req, res) {
+    let data = await itemManager.getTasks();
+    if (!data) {
+        data = [];
+    }
+    res.status(200).json(data);
+}
+
+async function addTask(req, res) {
+    if (!req.body.task || !req.body.date) {
+        const error = new Error("wrong parameters");
+        error.statusCode = 400;
+        throw error;
+    }
+    try{
+        await itemManager.addTask(req.body);
```

Comment:
instead of sending the entire body to the manager - send specific params
This is a best practice & will prevent unwanted parameter from infiltrating your code
This is the controller purpose